src/main/java/cn/nukkit/level/generator/biome/BeachBiome.java

```diff
@@ -1,28 +1,21 @@
 package cn.nukkit.level.generator.biome;
 
-import cn.nukkit.block.Block;
-import cn.nukkit.block.BlockSand;
-import cn.nukkit.block.BlockSandstone;
+import cn.nukkit.level.generator.populator.PopulatorSugarcane;
 
 /**
  * Author: PeratX
  * Nukkit Project
  */
 public class BeachBiome extends SandyBiome {
     public BeachBiome() {
-        //Todo: SugarCane
+        PopulatorSugarcane sugarcane = new PopulatorSugarcane();
+        sugarcane.setBaseAmount(0);
+        sugarcane.setRandomAmount(3);
+        this.addPopulator(sugarcane);
 
-        this.setElevation(62, 65);
+        this.setElevation(64, 66);
         this.temperature = 2;
```

Comment:
Sorry for do this thing, but real temperature and rainfall was float values, not double or integers.

Reply:
biome elevations, and the temperature/rainfall system need a full rework, they don't work nearly the same as in vanilla and generally pick stupid combinations

Reply:
Ok